### ISSUE TITLE
Convert checkout alterts to flashes 

### ISSUE THREAD

**Issue by tschumilas** [openfoodfoundation/openfoodnetwork#5655]
## Description
A user was stuck at checkout and reported she couldn't continue shopping or checkout,  and her screen was frozen.   I sent a series of screenshots to her showing the checkout error messages and 'update' button that appear when an item in the customer's cart goes out of stock.  She insisted she did not see these messages.   After back and forth we realized  her popup blocker was blocking these messages  - she couldn't see the messages and was stuck at checkout going nowhere.

We need to convert these alerts to flashes so that a popup blocker won't block them. 

## Acceptance Criteria & Tests

1.set up a pop up blocker
2. checkout from a store
3. While you are in checkout reduce the stock available of an item in your cart
4. See that you cannot see the system error message

**Comment by Erioldoesdesign** [comment]
Makes sense, doesn't need design since we already have a style for flashes messaging. 

It's actually really good UX to not have these popup messages that look more like the system messages, if people do see them and don't block, they are quite jarring and non-contextual to what's happening in the website UI.
